spec/services/subscriptions/free_trial_billing_service_spec.rb

```diff
@@ -5,8 +5,10 @@
 RSpec.describe Subscriptions::FreeTrialBillingService, type: :service do
   subject(:service) { described_class.new(timestamp:) }
 
+  before { travel_to timestamp }
```

Comment:
Curious about this one. Is it reseted  after the spec file is executed?

Reply:
I was expecting it to yes, but now I wonder if it was just because the `rails_helper` in my previous codebase was doing it 🤔 

In this case, I removed it because I think it was a left over of an initial version.